python/dgl/graphbolt/impl/fused_csc_sampling_graph.py

```diff
@@ -791,6 +791,32 @@ def sample_layer_neighbors(
             corresponding to each neighboring edge of a node. It must be a 1D
             floating-point or boolean tensor, with the number of elements
             equalling the total number of edges.
+        random_seed: torch.Tensor, optional
```

Comment:
Add an augment check to raise augment error before calling deeper to C++?

Reply:
What kind of a check, like checking whether the random_seed tensor is empty?

Reply:
To check whether the input is `An int64 tensor with one or two elements.` or not.

Reply:
same to seed2_contribution `A float value between [0, 1)`

Reply:
Added. Could you check and merge if the checks look good?